scripts/SubsetSelection/src/encoders/arctic.py

```
@@ -0,0 +1,229 @@
+            add_to_dataset=True
+        )
+        if encoder.rank == 0:
+            print("\nDataset results:")
+            print(embedded_ds)
+
+        # Also show an example of returning numpy arrays.
+        embeddings_np = encoder.encode(conversations, instruction="Retrieve relevant passages.", return_tensors=False)
+        if encoder.rank == 0:
+            print("\nNumpy array results:")
+            print(embeddings_np, embeddings_np.shape)
+    except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)